controller/app/validators/key_content_validator.rb

```diff
@@ -0,0 +1,15 @@
+class KeyContentValidator < ActiveModel::Validator
+   def validate(record)
+     key = Net::SSH::KeyFactory.load_data_public_key(record.type + " " + record.content)
```

Comment:
@sdodson Thank you for providing the updated code and additions to the test cases. 

One question: Looking at the exceptions handling, shouldn't the rescue line have the "NotImplementedError" as a parameter? If so, I believe it will catch everything - i.e.:
  'rescue NotImplementedError'